doc/releases/changelog-0.37.0.md

```diff
@@ -236,7 +236,7 @@ via the `NoiseModel` class and an `add_noise` transform.
   Finally, to modify a circuit mid-run, simply call the desired PennyLane operations:
 
   ```pycon
-  (pldb) qml.CNOT(wires=(0,2))
+  (pldb): qml.CNOT(wires=(0,2))
```

Comment:
```suggestion
  [pldb] qml.CNOT(wires=(0,2))
```
The PLDB prompt was changed in feature freeze. See [here](https://github.com/PennyLaneAI/pennylane/blob/9a6057492f70af415c3429a8bb159d7723022d04/pennylane/debugging/debugger.py#L46), this is from the RC branch.

I see that there are other instances of this in the changelog, could you update those as well?

Reply:
done